### PR TITLE
Updating the version of Standalone TCK's as part of Jakarta EE 9.1 release.

### PR DESCRIPTION
**Pull request by RohitKumarJain**
---
name: Pull Request
about: Create a pull request for a Platform TCK change
title: ''Updating the version of Standalone TCK's as part of Jakarta EE 9.1 release."
labels: ''
assignees: ''

---

**Fixes Issue**
https://github.com/eclipse-ee4j/jakartaee-tck/issues/633

**Test Link (CI)**
https://ci.eclipse.org/jakartaee-tck/job/rohit-jakartaee-tck/job/versionUpgrade/2/artifact/

**Describe the change**
This PR is for updating the version for standalone TCK's as part Jakarta EE 9.1 release.

CC @gurunrao @scottmarlow

### CODE REVIEW

**Review by gurunrao — APPROVED** [review]
LGTM!!!

File: release/tools/jaxrs.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 <project name="JAXRS" basedir="." default="build">
     
     <property name="deliverable.version"  value="3.0"/>
-    <property name="deliverable.tck.version"  value="3.0.0"/>
+    <property name="deliverable.tck.version"  value="3.0.1"/>
```
Comment (scottmarlow) [inline release/tools/jaxrs.xml:22]: Sorry, I should of mentioned that we already released [3.0.1 for EE 9 for a TCK challenge](https://download.eclipse.org/jakartaee/restful-ws/3.0/jakarta-restful-ws-tck-3.0.1.zip).

Please increment again to 3.0.2

Comment (gurunrao) [reply release/tools/jaxrs.xml:22]: @scottmarlow  - shall we merge of 90X branch changes to master? 
Compare - https://github.com/eclipse-ee4j/jakartaee-tck/compare/9.0.x

File: release/tools/servlet.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 	<import file="../../bin/xml/ts.common.props.xml"/>
 
 	<property name="deliverable.version"  value="5.0"/>
-        <property name="deliverable.tck.version"  value="5.0.0"/>
+        <property name="deliverable.tck.version"  value="5.0.1"/>
```
Comment (scottmarlow) [inline release/tools/servlet.xml:25]: Please update to 5.0.2 for servlet.

We already released 5.0.1 via https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip.  Servlet + JAX-RS were the only TCK challenges.  

We started https://github.com/eclipse-ee4j/jakartaee-tck/wiki/TCK-Challenge-processing-status for tracking details of further TCK challenges.

We also started the https://github.com/eclipse-ee4j/jakartaee-tck/projects/2 project for tracking TCK challenges across EE release.

File: release/tools/jaxrs.xml (hunk 1 above)
Comment (scottmarlow) [reply release/tools/jaxrs.xml:22]: @gurunrao yes and thank you!  I missed doing that before.  As per https://github.com/eclipse-ee4j/jakartaee-tck/projects/2, none of the challenged tests have been fixed yet so yes we should merge the 90x branch changes to master.

File: release/tools/servlet.xml (hunk 2 above)
Comment (RohitKumarJain) [reply release/tools/servlet.xml]: Done.

File: release/tools/jaxrs.xml (hunk 1 above)
Comment (RohitKumarJain) [reply release/tools/jaxrs.xml]: Done.

**Pull request merged**